x-pack/plugins/triggers_actions_ui/public/application/sections/alerts_table/alerts_table.tsx

```diff
@@ -36,15 +53,44 @@ const AlertsTable: React.FunctionComponent<AlertsTableProps> = (props: AlertsTab
     alertsTableConfiguration.columns.map(({ id }) => id)
   );
 
+  const leadingControlColumns = useMemo(() => {
+    return getLeadingControlColumns({
```

Comment:
The API for seems a bit odd, since we're passing through the leading control columns just to return it as a concatenated array, do you think that's worth abstracting out?

Reply:
Fair point. I typically like to see functional components as slim as possible but I might have overdone it here!